components/viz/service/BUILD.gn

```diff
@@ -255,6 +255,7 @@ viz_component("service") {
     "//services/viz/privileged/mojom",
     "//skia",
     "//skia:skcms",
+    "//starboard/build:starboard_buildflags",
```

Comment:
This should be also `if` gated for `is_cobalt`

Reply:
As this is used in `components/viz/service/display/overlay_processor_android.cc`, where we modified Chromium's existing codes, not adding new files. As a result, we need this flag to be global, not just `is_cobalt`.

We will have two cases to add `//starboard/build:starboard_buildflags`:
1. Modifying existing Chromium's code: add to the global deps
2. Adding new code (only for Cobalt): gated by `is_cobalt`

Reply:
Is this still the case if we have `BUILDFLAG(IS_COBALT)`/`BUILDFLAG(USE_STARBOARD)` in `//build/build_config.h`? Or can this be conditionally included now like Kaido suggested?

Reply:
We can conditionally include `//starboard/build:starboard_buildflags`, but this mean when we modify existing Chromium codes, we should use `BUILDFLAG(IS_COBALT)` / `BUILDFLAG(USE_STARBOARD)`, and `BUILDFLAG(USE_STARBOARD_MEDIA)` depends on them. As if `is_cobalt=false`, it won't recognize  `BUILDFLAG(USE_STARBOARD_MEDIA)`.

Reply:
For now, use
```
#if BUILDFLAG(IS_COBALT)
#if BUILDFLAG(USE_STARBOARD_MEDIA)
// Do work
#endif
#endif
```
and I'll work on changing this so it's easier to use. Thanks for bringing it up.

Reply:
Done.